frontend/.eslintrc.js

```diff
@@ -165,6 +165,26 @@ module.exports = {
     "lodash/prefer-is-nil": "off",
     "lodash/prefer-matches": "off",
     "lodash/path-style": "off",
+    "sort-imports": [
+      "error",
+      {
+        ignoreCase: true,
+        ignoreDeclarationSort: true,
+      },
+    ],
+    "import/order": [
+      1,
```

Comment:
Note: 
i have this set to warning in the case that it is unable to auto fix it, I don't want it to be blocking. Open to feedback here.

Reply:
Thats probably fair since we've gone this long without it 😂